.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - stable
```

Comment:
I'd specify LTS instead of stable https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions guessing they might be different

Reply:
Change it